### PR TITLE
AP_Notify: fix if statement

### PR DESCRIPTION
**Pull request by mirkix**
fix if statement

### CODE REVIEW

File: libraries/AP_Notify/ToneAlarm_Linux.cpp [hunk 1]
```diff
@@ -37,7 +37,7 @@ bool ToneAlarm_Linux::init()
 {
     // open the tone alarm device
     err = hal.util->toneAlarm_init();
```
Comment (jschall) [inline libraries/AP_Notify/ToneAlarm_Linux.cpp]: It'd be simpler to change this to
    err = !hal.util->toneAlarm_init();

Comment (mirkix) [inline libraries/AP_Notify/ToneAlarm_Linux.cpp]: Yes, thats better.

**Comment by tridge** [comment]
applied, thanks!
I squashed the two commits together using interactive rebase. It's worth learning to use interactive rebase btw - it makes it really easy to cleanup patches.

**Comment by mirkix** [comment]
I struggle a bit with GIT, because I am new to git, but it's getting better each day.

**Comment by tridge** [comment]
no worries! I'm happy to do bits of cleanup like this. I mentioned interactive rebase as it is quite fun to use once you learn it and is very useful.

**Pull request closed without merging**